examples/binding-scala/js/build.sbt

```diff
@@ -0,0 +1,13 @@
+enablePlugins(ScalaJSPlugin)
```

Comment:
Usually we create a root project that holds all assets and server-side code.
The root project may enable SbtJsEngine, PlayScala, or SbtWeb plugins.

For this TodoMVC application, I did not use PlayScala because there is no
server-side code, though I still kept the convention.



2016-08-02 20:39 GMT+08:00 Øyvind Raddum Berg <notifications@github.com>:

> In examples/binding-scala/js/build.sbt
> <https://github.com/tastejs/todomvc/pull/1615#discussion_r73146392>:
>
> > @@ -0,0 +1,13 @@
> > +enablePlugins(ScalaJSPlugin)
>
> Do we really need an sbt subproject here? Wouldnt it be simpler if the
> build only had the root project, and the scala source code would live in
> src/main/scala instead of under js/?
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub
> <https://github.com/tastejs/todomvc/pull/1615/files/43c87020b7e7cbf6b5f80bf4ac8998c157a9f9c0#r73146392>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/AAktus12ln468b4JIm0amOm1QrREnuHwks5qbzqLgaJpZM4IFv4z>
> .
>



-- 
杨博 (Yang Bo)